packages/common/src/logger.ts

```diff
@@ -18,7 +18,20 @@ export const serverHost = localBuildOrDev
   ? `https://${globalThis.process.env['VITE_SERVER_HOST']}:${globalThis.process.env['VITE_SERVER_PORT']}`
   : `https://${globalThis.process.env['VITE_SERVER_HOST']}`
 
+const disableLog = process.env.APP_ENV === 'test' || process.env.TEST || process.env.DISABLE_LOG
```

Comment:
should just be DISABLE_LOG, not TEST or APP_ENV = test

Reply:
k, fixed